### PR TITLE
[ i18n sprint ] Move translations from language specific lib-home-page templates to properties.

### PR DESCRIPTION
**Pull request by litvinovg**
[Companion PR](https://github.com/vivo-project/VIVO-languages/pull/115)
#What does this pull request do?#
Moved translations from language specific lib-home-page templates to properties.
Redesigned research statistic on home page.
#How to test#
For themes wilma, nemo, tenderfoot:
Open home page, view list of research types

### CODE REVIEW

**Review by chenejac — APPROVED** [review]

**Review by bkampe — APPROVED** [review]
Works like it should. 
Maybe the design/arrangement could be improved a little bit.

**Comment by litvinovg** [comment]
> Works like it should. Maybe the design/arrangement could be improved a little bit.

I will try to prepare a solution with a better look

**Review by bkampe — APPROVED** [review]
Works and is looking good.

File: webapp/src/main/webapp/templates/freemarker/lib/lib-home-page.ftl [hunk 1]
```diff
@@ -116,37 +116,60 @@
...
 <section id="home-research" class="home-sections">
     <h4>${i18n().research_capitalized}</h4>
-    <ul>
-        <#list classGroups as group>
+        <#if isResearchContentFound(classGroups) >
+        	<@printResearchTable classGroups />
+        <#else>
+            <p><li style="padding-left:1.2em">${i18n().no_research_content_found}</li></p>
+        </#if>
+    </ul>
```
Comment (chenejac) [inline webapp/src/main/webapp/templates/freemarker/lib/lib-home-page.ftl:126]: There is no opening tag for this?

**Review by chenejac — CHANGES_REQUESTED** [review]

File: webapp/src/main/webapp/templates/freemarker/lib/lib-home-page.ftl (hunk 1 above)
Comment (litvinovg) [reply webapp/src/main/webapp/templates/freemarker/lib/lib-home-page.ftl]: Removed closing tag. Thank you.

**Review by chenejac — APPROVED** [review]

**Pull request merged**